src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -48,6 +48,13 @@ namespace Microsoft.ML.Transforms
     /// <p>Supports inferencing of models in 1.2 and 1.3 format, using the
     /// <a href='https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime/'>Microsoft.ML.OnnxRuntime</a> library
     /// </p>
+    /// <p>Supports optional GPU execution via the CUDA 10.0. Requires
+    /// <a href='https://developer.nvidia.com/cuda-downloads'>CUDA 10.0 Toolkit</a>
+    /// and
+    /// <a href='https://developer.nvidia.com/cudnn'>cuDNN</a>
+    /// libraries need to be installed models can be scored in GPU. By default models are run on CPU. To execute on GPU,
+    /// set the parameter 'gpuDeviceID' to a valid non-negative number.
```

Comment:
>valid non-negative number [](start = 45, length = 25)

what does valid-non negative mean? Are the GPUs available in the system referenced by some sort of load index?